massa-execution/src/sce_ledger.rs

```diff
@@ -40,6 +46,152 @@ impl SCELedgerEntry {
     }
 }
 
+impl SerializeCompact for SCELedgerEntry {
+    fn to_bytes_compact(&self) -> Result<Vec<u8>, massa_models::ModelsError> {
+        let mut res: Vec<u8> = Vec::new();
+
+        // write balance
+        res.extend(self.balance.to_bytes_compact()?);
+
+        // write opt module data
+        if let Some(module_data) = &self.opt_module {
+            // write that it is present
+            res.push(1);
+
+            // write length
+            let length: u32 = module_data.len().try_into().map_err(|_| {
+                ModelsError::SerializeError(
+                    "SCE ledger entry module data too long for serialization".into(),
+                )
+            })?;
+            // TODO check against max length
```

Comment:
A max len that should be added to serialization context ?

Reply:
We should create a discussion or an issue. *detailed of course* :bear:

Reply:
I guess it's related to #1200

Reply:
yes I think